conanfile.py

```diff
@@ -70,10 +88,10 @@ def build(self):
             vccmd = tools.vcvars_command(self.settings) + " &&"
             compiler_cc = '$PWD/compile cl -nologo'
             compiler_cxx = compiler_cc
-            
+
         elif self.settings.compiler == 'gcc':
             vccmd = ''
-            if 'MINGW_HOME' in os.environ: 
+            if 'MINGW_HOME' in os.environ:
                 compiler_cc = tools.unix_path(self.deps_env_info["mingw_installer"].CC)

```

Comment:
probably this is unnecessary, as mingw_installer already sets CC/CXX

Reply:
Good point.

Reply:
I've removed unnecessary handling of CC